lxd/seccomp/seccomp.go

```diff
@@ -1316,12 +1338,25 @@ func (s *Server) doDeviceSyscall(c Instance, args *MknodArgs, siov *Iovec) int {
 	dev["mode_t"] = fmt.Sprintf("%d", args.cMode)
 	dev["dev_t"] = fmt.Sprintf("%d", args.cDev)
 
+	// has CAP_MKNOD capability?
+	hasCapability, err := isCapableInCtInitUserns(siov, C.CAP_MKNOD)
+	if err != nil {
+		logger.Errorf("%v", err)
```

Comment:
We shouldnt use context-less logging entries like this as they are very tricky to use when a LXD installations has lots of instances.

As you've been supplied the `Instance` as `c` in this function, please can all log entries include the instance name and project name, e.g. like

```go
l := logger.AddContext(logger.Ctx{"project": c.Project().Name, "instance": c.Name()})
l.Error()
```

Reply:
Fixed, thanks for suggestion!